int16 R
int16 G
int16 B